kafka-client/src/main/java/dev/responsive/kafka/internal/stores/CommitBuffer.java

```diff
@@ -158,10 +201,71 @@ static <K, S extends RemoteTable<K>> CommitBuffer<K, S> from(
     this.maxBatchSize = maxBatchSize;
     this.subPartitioner = subPartitioner;
     this.clock = clock;
+    this.lastFlush = clock.get();
 
     logPrefix = String.format("commit-buffer [%s-%d] ", table.name(), changelog.partition());
     log = new LogContext(logPrefix).logger(CommitBuffer.class);
 
+    lastFlushMetric = metrics.metricName(
+        TIME_SINCE_LAST_FLUSH,
+        TIME_SINCE_LAST_FLUSH_DESCRIPTION,
+        metrics.storeLevelMetric(metrics.computeThreadId(), changelog, storeName)
+    );
+    metrics.addMetric(
+        lastFlushMetric,
+        (Gauge<Long>) (config, now) -> now - lastFlush.toEpochMilli()
+    );
+
+    flushesSensor = metrics.addSensor(FLUSH);
```

Comment:
this name should be unique to this store, otherwise you'll get a single sensor for all stores and mix up samples

Reply:
Ah, good catch. I'm still adding unit tests for these metrics, I'll make sure to have coverage for this case